packages/platform/platform-mcp/src/decorators/prompt.spec.ts

```
@@ -0,0 +1,30 @@
+import {DITest, inject, Injectable} from "@tsed/di";
+import {Description, Title} from "@tsed/schema";
+
+import {Prompt} from "../..";
+
+@Injectable()
+export class TestPrompt {
+  @Prompt()
+  @Title("Title")
+  @Description("Description")
+  prompt() {}
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)